src/PrestaShopBundle/EventListener/DemoModeEnabledListener.php

```diff
@@ -54,7 +55,7 @@ class DemoModeEnabledListener
     private $translator;
 
     /**
-     * @var SessionInterface
```

Comment:
Could we have SessionInterface ?
If you tell me we cannot, we keep Session 😉

Reply:
We can't because we use methods not in the interface, but only in the Session class